Sources/SignalProducer.swift

```diff
@@ -100,7 +100,7 @@ public struct SignalProducer<Value, Error: Swift.Error> {
 	/// - parameters:
 	///   - values: A sequence of values that a `Signal` will send as separate
```

Comment:
It doesn't feel like the type checker could ever cause any chaos here though.

`SignalProducer` is a discrete time sequence, while `Sequence` is immediate. This means we would likely never conform `SignalProducer` to `Sequence`, or in other words the two signatures would never collide in any sense.